jinahub/indexers/storage/PostgreSQLStorage/__init__.py

```diff
@@ -36,11 +36,13 @@ def __init__(
         table: str = 'default_table',
         max_connections=5,
         default_traversal_paths: List[str] = ['r'],
+        embedding_dtype: type = None,
```

Comment:
add a default of float.64

Reply:
if it is None, then it will be the same as before. If default of `dtype` in np.frombuffer is None.